simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -1728,4 +1763,10 @@ class where_expression<simd_mask<std::uint64_t, simd_abi::avx2_fixed_size<4>>,
 }  // namespace Experimental
 }  // namespace Kokkos
 
+#if defined(__HIPCC__) &&       \
+    ((HIP_VERSION_MAJOR > 5) || \
+     ((HIP_VERSION_MAJOR == 5) && (HIP_VERSION_MINOR >= 6)))
```

Comment:
```suggestion
#if defined(KOKKOS_IMPL_WORKAROUND_ROCM_AVX2_ISSUE)
```

Reply:
I just undefined it unconditionally now.